replicator-resources.qmd

```diff
@@ -309,6 +309,18 @@ You can try the web-based uploader which will take any URL directly:
 
 ![](img/nuvolos-large-upload.png)
 
+When copying the dropbox URL, remember to change 'dl=0' to 'dl=1' inside the URL. This forces the download. For example:
+```
+https://www.dropbox.com/scl/fo/gumryf1zs8lwk5zq5udo7/ABrnRWk5w859IIibD-_QM1Q/Szumilo-20230708-R1?*dl=1*&preview=Niko+Szumilo+-+MS20230708-replication.zip&rlkey=ei2aozcysqcchaoiwwanjcg4s&subfolder_nav_tracking=1
```

Comment:
let's convert this into
```
https://www.dropbox.com/scl/fo/gumryf1zs8lwk5zq5udo7/ABrnRWk5w859IIib/Author-YYYYMMDD-R1?*dl=1*
```

I garbled up the URL and deleted the rest of the parameters which we don't need.